swim/join_test.go

```diff
@@ -92,20 +95,20 @@ func (s *JoinSenderTestSuite) TestSelectMultipleGroups() {
 }
 
 func (s *JoinSenderTestSuite) TestSelectGroupExcludes() {
-	seedBootstrapHosts(s.node, genAddresses(1, 1, 5))
+	seedBootstrapHosts(s.node, fakeAddresses(1, 1, 5))
 
 	joiner, err := newJoinSender(s.node, nil)
 	s.Require().NoError(err, "cannot have an error")
 	s.Require().NotNil(joiner, "joiner cannot be nil")
 
-	group := sort.StringSlice(joiner.SelectGroup([]string{"127.0.0.1:3003", "127.0.0.1:3005"}))
+	group := sort.StringSlice(joiner.SelectGroup([]string{"192.0.2.1:3", "192.0.2.1:5"}))
 	group.Sort()
 
-	s.EqualValues([]string{"127.0.0.1:3002", "127.0.0.1:3004"}, group)
+	s.EqualValues([]string{"192.0.2.1:2", "192.0.2.1:4"}, group)
```

Comment:
I don't think we need to refer to the comment in every place an IP address is used. It is an IP address, the same as 127.0.0.1 or any other IP address.

The question here is surrounding the patch/change and the reasoning for this change is described in the commit message.